ml-prims/src/hmm/cuda/cublas_wrappers.h

```diff
@@ -0,0 +1,79 @@
+#pragma once
+
```

Comment:
General comment for this whole file:
I think the contents of this file belong to ml-prims/src/linalg/cublas_wrappers.h